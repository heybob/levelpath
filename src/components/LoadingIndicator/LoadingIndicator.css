.loading-bkg {
    display: flex;
    width: 100%;
    height: 100vh;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    z-index: 100;
    position: absolute;
    top: 0;
}
.loading-content {
    width: 200px;
    border-radius: 1rem;
    background-color: white;
    padding: 1rem;
}