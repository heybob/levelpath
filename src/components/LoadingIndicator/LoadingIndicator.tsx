import './LoadingIndicator.css'
import {Text} from '../../shared/constants';

export function LoadingIndicator() {
    return (
        <div className='loading-bkg'>
            <div className='loading-content'>{Text.loadingData}</div>
        </div>
        );
}