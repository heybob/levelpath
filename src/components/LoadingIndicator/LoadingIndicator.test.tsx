import renderer from 'react-test-renderer';
import { LoadingIndicator } from "./LoadingIndicator";

describe('Testing Loading Indicator Component', () => {
  it('It should render loading indicator', () => {
      const tree = renderer.create(<LoadingIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
