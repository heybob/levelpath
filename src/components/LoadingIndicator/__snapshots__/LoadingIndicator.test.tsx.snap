// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Loading Indicator Component It should render loading indicator 1`] = `
<div
  className="loading-bkg"
>
  <div
    className="loading-content"
  >
    Loading Data...
  </div>
</div>
`;
